### PR TITLE
Removal of Contagion.

### PR DESCRIPTION
**Pull request by SticksDeveloper**
### Removes Contagion Entities | Armor.

### CODE REVIEW

File: src/main/java/com/paneedah/mwc/wearables/Vests.java [hunk 1]
```diff
@@ -211,49 +211,46 @@ public static void init(Object mod, CompatibleFmlPreInitializationEvent event) {
...
+//            GL11.glRotatef(0F, 0f, 0f, 1f);
+//        })
+//        .withInventoryPositioning(stack -> {
+//	        new Transform()
+//	        .withPosition(-0.1, -4, -0.25)
+//	        .withRotation(-15, 160, 5)
+//	        .withScale(3, 3, 3)
+//	        .doGLDirect();
+//        })
+//        .build(ModernWarfareMod.MOD_CONTEXT);
```
Comment (Desoroxxx) [inline src/main/java/com/paneedah/mwc/wearables/Vests.java:254]: Why? I would not consider real life armor to be contagion, I personally think they should stay, correct me if I wrong.

What do you think @Paneedah

File: src/main/java/com/paneedah/mwc/wearables/Backpacks.java [hunk 2]
```diff
@@ -315,93 +315,93 @@ public static void init(Object mod, CompatibleFmlPreInitializationEvent event) {
...
+//            GL11.glTranslatef(0f, 0f, 0f);
+//            GL11.glRotatef(0F, 0f, 0f, 1f);
+//        })
+//        .withInventoryPositioning(stack -> {
+//            GL11.glScalef(1.1f, 1.1f, 1.1f);
+//            GL11.glTranslatef(-0.2f, -0.5f, 0f);
+//            GL11.glRotatef(-33F, 0f, 1f, 0f);
+//            GL11.glRotatef(20F, 1f, 0f, 0f);
+//        })
+//        .build(ModernWarfareMod.MOD_CONTEXT);
```
Comment (Desoroxxx) [inline src/main/java/com/paneedah/mwc/wearables/Backpacks.java:404]: I do not remember what these are, do you know if we should remove them with Contagion @Paneedah

File: src/main/java/com/paneedah/mwc/wearables/Backpacks.java [hunk 3]
```diff
@@ -43,27 +43,27 @@ public class Backpacks {
...
+//            GL11.glTranslatef(0f, 0f, 0f);
+//            GL11.glRotatef(0F, 0f, 0f, 1f);
+//        })
+//        .withInventoryPositioning(stack -> {
+//            GL11.glScalef(1.3f, 1.3f, 1.3f);
+//            GL11.glTranslatef(0.2f, -0.55f, 0f);
+//            GL11.glRotatef(23F, 0f, 1f, 0f);
+//            GL11.glRotatef(20F, 0f, 0f, 1f);
+//        })
+//        .build(ModernWarfareMod.MOD_CONTEXT);
```
Comment (Desoroxxx) [inline src/main/java/com/paneedah/mwc/wearables/Backpacks.java:66]: Same, I may be mistaken but this is not contagion related.

File: src/main/java/com/paneedah/mwc/proxies/ClientProxy.java [hunk 4]
```diff
@@ -62,12 +62,12 @@ public void init(final ModernWarfareMod mod, final CompatibleFmlInitializationEv
...
-
-        modelMesher.register(Armors.SCPGuardboots, 0, new ModelResourceLocation(ModReference.id + ":swat_boots", "inventory"));
-        modelMesher.register(Armors.SCPGuardchest, 0, new ModelResourceLocation(ModReference.id + ":scpguard_chest", "inventory"));
-        modelMesher.register(Armors.SCPGuardhelmet, 0, new ModelResourceLocation(ModReference.id + ":scpguard_helmet", "inventory"));
+//        modelMesher.register(Armors.Dclassboots, 0, new ModelResourceLocation(ModReference.id + ":dclass_boots", "inventory"));
+//        modelMesher.register(Armors.Dclasschest, 0, new ModelResourceLocation(ModReference.id + ":dclass_chest", "inventory"));
+//
+//        modelMesher.register(Armors.SCPGuardboots, 0, new ModelResourceLocation(ModReference.id + ":swat_boots", "inventory"));
+//        modelMesher.register(Armors.SCPGuardchest, 0, new ModelResourceLocation(ModReference.id + ":scpguard_chest", "inventory"));
+//        modelMesher.register(Armors.SCPGuardhelmet, 0, new ModelResourceLocation(ModReference.id + ":scpguard_helmet", "inventory"));
```
Comment (Desoroxxx) [inline src/main/java/com/paneedah/mwc/proxies/ClientProxy.java:70]: Just remove it entirely

File: src/main/java/com/paneedah/mwc/proxies/ClientProxy.java [hunk 5]
```diff
@@ -111,9 +111,9 @@ public void init(final ModernWarfareMod mod, final CompatibleFmlInitializationEv
         modelMesher.register(Armors.Juggernautchest, 0, new ModelResourceLocation(ModReference.id + ":juggernaut_chest", "inventory"));
         modelMesher.register(Armors.Juggernauthelmet, 0, new ModelResourceLocation(ModReference.id + ":juggernaut_helmet", "inventory"));
 
-        modelMesher.register(Armors.Astronautboots, 0, new ModelResourceLocation(ModReference.id + ":Astronaut_boots", "inventory"));
-        modelMesher.register(Armors.Astronautchest, 0, new ModelResourceLocation(ModReference.id + ":Astronaut_chest", "inventory"));
-        modelMesher.register(Armors.Astronauthelmet, 0, new ModelResourceLocation(ModReference.id + ":Astronaut_helmet", "inventory"));
+//        modelMesher.register(Armors.Astronautboots, 0, new ModelResourceLocation(ModReference.id + ":Astronaut_boots", "inventory"));
+//        modelMesher.register(Armors.Astronautchest, 0, new ModelResourceLocation(ModReference.id + ":Astronaut_chest", "inventory"));
+//        modelMesher.register(Armors.Astronauthelmet, 0, new ModelResourceLocation(ModReference.id + ":Astronaut_helmet", "inventory"));
```
Comment (Desoroxxx) [inline src/main/java/com/paneedah/mwc/proxies/ClientProxy.java:116]: Just remove it entirely

File: src/main/java/com/paneedah/mwc/wearables/Armors.java [hunk 6]
```diff
@@ -271,37 +271,37 @@ public static void init(Object mod, CompatibleFmlPreInitializationEvent event, M
...
+//        Builder SCPGuardArmorBuilder = new CustomArmor.Builder().withMaterial(Armors.Marine)
+//                .withUnlocalizedName("SCPGuard")
+//                .withTextureName("scpguard")
+//                .withModelClass("com.paneedah.mwc.models.SCPGuard")
+//                .withHudTextureName("Marine")
+//                .withCreativeTab(ModernWarfareMod.ArmorTab);
+//
+//        Armors.SCPGuardhelmet = SCPGuardArmorBuilder.buildHelmet(modContext);
+//        Armors.SCPGuardchest = SCPGuardArmorBuilder.buildChest(modContext.isClient());
+//        Armors.SCPGuardboots = SCPGuardArmorBuilder.buildBoots(modContext.isClient());
```
Comment (Desoroxxx) [inline src/main/java/com/paneedah/mwc/wearables/Armors.java:305]: Just remove it entirely

File: src/main/java/com/paneedah/mwc/entities/Entities.java [hunk 7]
```diff
@@ -428,154 +428,154 @@ public static void init(CommonProxy commonProxy) {
...
+//        .withAiTargetTask(3, e -> compatibility.createAINearestAttackableTarget(e, EntityPigZombie.class, true))
+//        .withAiTargetTask(3, e -> compatibility.createAINearestAttackableTarget(e, EntitySpellcasterIllager.class, true))
+//        .withAiTargetTask(3, e -> compatibility.createAINearestAttackableTarget(e, EntityStray.class, true))
+//        .withAiTargetTask(3, e -> compatibility.createAINearestAttackableTarget(e, EntityVindicator.class, true))
+//        .withAiTargetTask(3, e -> compatibility.createAINearestAttackableTarget(e, EntityWitch.class, true))
+//        .withAiTargetTask(3, e -> compatibility.createAINearestAttackableTarget(e, EntityZombieVillager.class, true))
+//        .withAiTargetTask(4, e -> new BetterAINearestAttackableTarget<>((EntityCreature) e, EntityCustomMob.class, "soldier", true))
+//        .withAiTargetTask(4, e -> new BetterAINearestAttackableTarget<>((EntityCreature) e, EntityCustomMob.class, "terrorist", true))
+//        .withAiTargetTask(4, e -> new BetterAINearestAttackableTarget<>((EntityCreature) e, EntityCustomMob.class, "tyke", true))
+//        .register(ModernWarfareMod.MOD_CONTEXT);
```
Comment (Desoroxxx) [inline src/main/java/com/paneedah/mwc/entities/Entities.java:579]: Same, not really Contagion related, just a normal looking turret

File: src/main/java/com/paneedah/mwc/entities/Entities.java [hunk 8]
```diff
@@ -24,358 +24,358 @@ public static void init(CommonProxy commonProxy) {
...
+//        .withAiTask(3, e -> new CompatibleEntityAIAttackOnCollide((EntityCustomMob)e, EntityPlayer.class, 1.0D, false))
+//        .withAiTask(5, e -> new EntityAIWander((EntityCreature)e, 1.0D))
+//        .withAiTask(6, e -> new EntityAIWatchClosest(e, EntityPlayer.class, 50.0F))
+//        .withAiTask(6, e -> new EntityAILookIdle(e))
+//        .withAiTargetTask(1, e -> new EntityAIHurtByTarget((EntityCreature)e, false))
+//        .withAiTargetTask(2, e -> compatibility.createAINearestAttackableTarget(e, EntityPlayer.class, true))
+//        .withAiTargetTask(3, e -> compatibility.createAINearestAttackableTarget(e, EntityVillager.class, true))
+////   .withCollisionAttack(new InfectionAttack(0.1f, 20000, 1.2f, 1.0f, 1.8f))
+////      .withDelayedAttack(new ExplosionAttack(2f, true, 30))
+//        .register(ModernWarfareMod.MOD_CONTEXT);
```
Comment (Desoroxxx) [inline src/main/java/com/paneedah/mwc/entities/Entities.java:378]: Just remove it entirely

**Review by Desoroxxx — CHANGES_REQUESTED** [review]

File: src/main/java/com/paneedah/mwc/wearables/Backpacks.java (hunk 3 above)
Comment (SticksDeveloper) [reply src/main/java/com/paneedah/mwc/wearables/Backpacks.java:66]: this should've been with the "MASTER" class push instead but made a mistake. but It still needs to get removed

File: src/main/java/com/paneedah/mwc/wearables/Armors.java (hunk 6 above)
Comment (SticksDeveloper) [reply src/main/java/com/paneedah/mwc/wearables/Armors.java:305]: Removed

File: src/main/java/com/paneedah/mwc/proxies/ClientProxy.java (hunk 5 above)
Comment (SticksDeveloper) [reply src/main/java/com/paneedah/mwc/proxies/ClientProxy.java:116]: Removed

File: src/main/java/com/paneedah/mwc/wearables/Backpacks.java (hunk 3 above)
Comment (SticksDeveloper) [reply src/main/java/com/paneedah/mwc/wearables/Backpacks.java:66]: It was supposed to be in Master Branch but I messed up but It's still broken either way.

File: src/main/java/com/paneedah/mwc/wearables/Backpacks.java (hunk 2 above)
Comment (SticksDeveloper) [reply src/main/java/com/paneedah/mwc/wearables/Backpacks.java:404]: It's an old backpack which is broken that was under the MISC Default Minecraft Creative Tab

**Pull request merged**